### DISCUSSION TITLE
ImageDataGenerator

### DISCUSSION THREAD

**Discussion by virupakshshinde-blip** [Niketkumardheeryan/ML-CaPsule#1299] [Q&A] [answered]
How to use certain type of pipeline for normalization or augmentation techniques in ImageDataGenerator

**Comment by Mahatva1303** [comment, marked as answer]
from tensorflow.keras.preprocessing.image import ImageDataGenerator

# Normalization pipeline
train_datagen = ImageDataGenerator(
    rescale=1./255
)

# Augmentation pipeline
augmented_datagen = ImageDataGenerator(
    rescale=1./255,
    rotation_range=20,
    zoom_range=0.2,
    horizontal_flip=True,
    shear_range=0.2
)

**Reply by Mahatva1303** [reply]
@virupakshshinde-blip hope this answers your query if yes please mark question as answered
